requirements_dev.txt

```diff
@@ -1,5 +1,7 @@
 # Install with -U to keep all requirements up-to-date
 # code linting and formatting
+# Black 22.1 has a fatal bug, so use the latest previous for now
+black==21.12b0
```

Comment:
I first tried `black!=22.1`, which I think **would've** worked, except Black has **never** had a "stable" release until now. They are all beta, so if we exclude the one and only stable release, `pip` refuses to install any of the **unstable** releases. Ironically, the first ever "stable" release is fatally broken. 🙄🤣

Prior to this change, 22.1 was giving the same error on every file it processed:
```
linting run-test: commands[0] | flake8
./setup.py:0:1: BLK999 Unexpected exception: unsupported operand type(s) for /: 'tuple' and 'str'
./util.py:0:1: BLK999 Unexpected exception: unsupported operand type(s) for /: 'tuple' and 'str'
./plugins/sqlfluff-templater-dbt/setup.py:0:1: BLK999 Unexpected exception: unsupported operand type(s) for /: 'tuple' and 'str'
./plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py:0:1: BLK999 Unexpected exception: unsupported operand type(s) for /: 'tuple' and 'str'
./plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/__init__.py:0:1: BLK999 Unexpected exception: unsupported operand type(s) for /: 'tuple' and 'str'
./plugins/sqlfluff-templater-dbt/test/templater_test.py:0:1: BLK999 Unexpected exception: unsupported operand type(s) for /: 'tuple' and 'str'
./plugins/sqlfluff-templater-dbt/test/rules_test.py:0:1: BLK999 Unexpected exception: unsupported operand type(s) for /: 'tuple' and 'str'
```